portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -83,9 +89,10 @@ private static String getParameter(HttpServletRequest request, String name, Stri
     return value;
   }
 
-  private static void storeComments(String name, String comment, long timestamp) {
+  private static void storeComments(String name, String email, String comment, long timestamp) {
```

Comment:
You may consider to use builder instead of a function in the future:
https://www.informit.com/articles/article.aspx?p=1216151&seqNum=2

No need to change in this PR.

Reply:
Oh, I didn't know about these pros and cons before, thanks.